internal/website/docusaurus.config.js

```diff
@@ -103,7 +103,7 @@ module.exports = {
           sidebarPath: require.resolve('./sidebars.js'),
           // Please change this to your repo.
           editUrl:
-            'https://github.com/wpengine/faustjs/edit/docs-setup-docusaurus/internal/website/',
+            'https://github.com/wpengine/faustjs/edit/canary/internal/website/',
```

Comment:
Maybe change this to `site-dev` so change suggestions make it to the staging site first.

Reply:
Ohh, yeah. Good call